bodhi/tests/server/functional/test_updates.py

```diff
@@ -453,7 +453,7 @@ def test_list_updates(self):
         self.assertEquals(up['locked'], True)
         self.assertEquals(up['alias'], alias)
         self.assertEquals(up['karma'], 1)
-        self.assertEquals(up['url'], '%s/updates/%s' % (baseurl, alias))
+        self.assertEquals(up['url'], (urlparse.urljoin(config['base_address'], '/updates/%s' % alias)))
```

Comment:
I've run a few times into issues using ``urlparse.urljoin()`` so I now try to avoid it in favor of a construct such as:

``
'%s/updates/%s' % (baseurl.rstrip('/'), alias))
``
Close to the original one but ensure there are no `//`.